tools/build_and_test_onnxrt.sh

```diff
@@ -22,7 +22,7 @@
 # THE SOFTWARE.
 #####################################################################################
 cd /onnxruntime
-pip3 install -r requirements.txt
+pip3 install -r requirements-dev.txt
```

Comment:
I don't think this file exists in /onnxruntime/requirements-dev.txt.  The Dockerfile copies it to /requirements-dev.txt.  But isn't this suppose to be the ORT requirements?  If no, and it is suppose to be the migx requirements then a /requirements-dev.txt would work.

Reply:
oh nevermind I see the ONNX based requirements-dev.txt file.